cdap-ui/app/directives/plumb/my-plumb-factory.js

```diff
@@ -16,11 +16,11 @@ angular.module(PKG.name + '.commons')
       maxConnections: -1,
       paintStyle: {
         strokeStyle: 'white',
-        fillStyle: '#666',
+        fillStyle: '#666e82',
```

Comment:
I don't believe we can use the variable defined in a less file in javascript file. We don't ship less files right.. Any suggestions to do it in a straight forward way apart from querying through css selectors ?